packages/app/obojobo-document-engine/src/scripts/oboeditor/overwrite-bug-fixes.js

```diff
@@ -131,4 +131,29 @@ Transforms.mergeNodes = (editor, options) => Editor.withoutNormalizing(editor, (
 	if (emptyRef) {
 		emptyRef.unref()
 	}
-})
\ No newline at end of file
+})
+
+// This one is directly copied from Slate's master branch, but it has not been released yet
```

Comment:
I'm thinking we may want to throw an error if Transforms.setPoint is already set. That way in the future when we upgrade we'll remember this bit of code and be able to delete it.

Reply:
Transforms.setPoint is already set, but the current release has a bug where it doesn't alter the selection properly.  This code is taken straight from the master branch to fix that but has not been released in a version.

Reply:
Ah okay, I jumped to a conclusion that Transforms.setPoint was new. Is there a github issue or page that we could then refer to in the comments? Something so that later on when re-reviewing this code we'd be able to jump to that linked page and see if its been fixed.